pkg/virt-handler/vm.go

```diff
@@ -233,7 +233,7 @@ func handleDomainNotifyPipe(domainPipeStopChan chan struct{}, ln net.Listener, v
 					defer conn.Close()
 
 					log.Log.Object(vmi).Infof("Accepted new notify pipe connection for vmi")
-					copyErr := make(chan error)
+					copyErr := make(chan error, 2)
```

Comment:
I think that the race detector of golang should have detected that. Seems like we miss enabling it when we run the tests with code coverage. We need to fix that in a follow-up.